opdracht-1/styles.css

```diff
@@ -0,0 +1,54 @@
+/* ======================================================
+   OEFENEN MET CSS SELECTORS
+========================================================= */
+
+/* 0: Voorbeeld: maak de achtergrondkleur van de pagina wit */
+
+body{
+    background-color: #ffffff;
+}
+/* 1.1 Geef de 'img' tag in the header een ID en zet zijn breedte op 100% */
+#Tutti{
+    width: 100%;
+}
+/* 1.2 Verander de kleur van alle list items in de unordered list naar hsl(110, 29%, 52%) */
+ul li{
+    color: hsl(110, 29%, 52%);
+}
+/* 1.3 Geef de 'span' tag in de footer een beschrijvende class name en maak de lettergrootte kleiner  */
+span{
+    font-size: 10px;
+}
+/* 1.4 Haal de text-decoration van navigatie-links weg, geef onbezochte links de kleur hsl(359, 79%, 56%) en verander de kleur wanneer de gebruiker er overheen hovert. */
+a{
+    text-decoration: none;
+}
+a:link{
+    color: hsl(359, 79%, 56%);
+}
+a:hover{
+    color: hsl(205, 50%, 42%);
+}
+/* 1.5 Maak een hover- en visited stijl voor alle links in 'main'. */
```

Comment:
Omdat je de main links geen originele kleur heb meegegeven worden ze allemaal één en dezelfde kleur na het bezoeken ervan.

Reply:
De grap is dat dit alleen gebeurd in de main omdat in het html bestand de links allemaal naar dezelfde # refereren, als je deze veranderd in bijvoorbeeld een ! en een & bij de andere 2 dan gebeurd dit niet

Reply:
Technical difficulties.